content/api/stats.md

```diff
@@ -0,0 +1,167 @@
+  ],
+  "parents": [], // Parent chunk IDs? Names?
+  "rendered": true, // ?
+  "size": 188057 // Chunk size in bytes (?)
+}
+```
+
+The `chunks` object will also contain a list of `origins` describing how the given chunk originated. Each `origins` object follows the following schema:
+
+``` js
+{
+  "loc": "", // Lines of code that generated this chunk (?)
+  "module": "(webpack)\\test\\browsertest\\lib\\index.web.js", // ?
+  "moduleId": 0, // The ID of the module
+  "moduleIdentifier": "(webpack)\\test\\browsertest\\lib\\index.web.js", // ?
+  "moduleName": "./lib/index.web.js", // ?
+  "name": "main", // ?
+  "reasons": [
+    // A list of the same `reasons` found in [module objects](#module-objects)
+  ]
+}
+```
+
+
+### Module Objects
+
+What good would these statistics be without some description of the compiled application's actual modules? Each module in the dependency graph is represented by the following structure:
+
+``` js
+{
+  "assets": [], // A list of [asset objects](#asset-objects) (?)
+  "built": true, // Maybe this means compiled, loaded or output (?)
+  "cacheable": true, // Whether or not this module is cacheable (?)
+  "chunks": [
+    // IDs of chunks that contain this module (?)
+  ],
+  "errors": 0, // Number of errors when resolving or processing the module (?)
+  "failed": false, // Whether or not compilation failed on this module (?)
+  "id": 0, // The ID of the module
+  "identifier": "(webpack)\\test\\browsertest\\lib\\index.web.js", // Another ID ?
+  "name": "./lib/index.web.js", // Path to the actual file
+  "optional": false, // ?
+  "prefetched": false, // Whether or not the module was prefetched (a build optimization)
+  "profile": {
+    // Module specific compilation stats corresponding to the [`--profile` flag](/api/cli#profiling)
+    "building": 73,
+    "dependencies": 242,
+    "factory": 11
+  },
+  "reasons": [
+    // See the description below...
+  ],
+  "size": 3593, // Module size in bytes?
+  "source": "// Should not break it...\r\nif(typeof...", // The stringified raw source (?)
+  "warnings": 0 // Number of warnings when resolving or processing the module (?)
+}
+```
+
+Every module also contains a list of `reasons` objects describing why that module was included in the dependency graph. Each "reason" is very similar to the `origins` objects seen above in the [chunk objects](#chunk-objects) section:
+
+``` js
+{
+  "loc": "33:24-93", // Lines of code that caused the module to be included
+  "module": "./lib/index.web.js", // ?
+  "moduleId": 0, // The ID of the module
+  "moduleIdentifier": "(webpack)\\test\\browsertest\\lib\\index.web.js", // ?
+  "moduleName": "./lib/index.web.js", // ?
+  "type": "require.context", // The [type of request](/api/module-methods) used
+  "userRequest": "../../cases" // ?
+}
+```
+
+
+### Errors and Warnings
+
+The `errors` and `warnings` properties each contain a list of strings. Each string contains a message and stack trace:
```

Comment:
stacktrace is only included when `errorDetails: true` was passed to `toJson` (that the default)